src/definitions/behaviors/form.js

```diff
@@ -795,6 +795,16 @@ $.fn.form = function(parameters) {
 
         },
 
+        can: {
+            useElement: function(element){
+               if ($.fn[element] !== undefined) {
+                   return true;
+               }
+               module.error(error.noElement.replace(/{element}/,element));
```

Comment:
I think costly RegExp is not required here.

```suggestion
               module.error(error.noElement.replace("{element}",element));
```

Reply:
Very good catch! Thx 👍